lib/winston-logstash.js

```diff
@@ -166,7 +166,7 @@ Logstash.prototype.connect = function () {
 
     self.socket = null;
 
-    if (!ECONNREFUSED_REGEXP.test(err.message)) {
+    if (err.message.indexOf('E') !== 0) {
```

Comment:
This seems a bit odd... Is the source of these errors such that "starting with E" is sufficiently specific not to catch some user-created error that starts with E?